src/Tinkoff.ISA.Scheduler/Program.cs

```diff
@@ -47,8 +48,11 @@ private static void Main()
                 .Configure<JiraSettings>(configuration.GetSection("JiraSettings"))
```

Comment:
we have to use new mechanism here. To test our first provider

Reply:
If I'm not mistaken, we have agreed to include it here, when all providers will be implemented according to the new interface

Reply:
@vmayushan @lesley29 what you think maybe we can start testing this mechanism in Jira integration?